src/index.js

```diff
@@ -9,19 +9,26 @@ import { Provider } from "react-redux";
 import Header from "./components/Header";
 import { Footer } from "./components/Footer";
 import "./reset.css";
+import { createTheme, ThemeProvider } from "@mui/material";
 
 const store = configureStore();
 
-window.React = React;
```

Comment:
wtf it was?

Reply:
Already removed. Tried to fix react dev tools